### PR TITLE
Use a `Set` and ES2015ify

### LINKED ISSUE #1: Use a Set
Should be faster.

Though Set in current Node V8 is just too [buggy](https://github.com/sindresorhus/array-uniq/blob/d5e311f37692dfd25ec216490df10632ce5f69f3/index.js#L29-L50) and slow to be worth it right now. Maybe someday. Could use a polyfill, but the ones I found are bloated or override the built-ins.

### PR DESCRIPTION
**Pull request by kevva**
Fixes #1.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
 
-binaryExtensions.forEach(function (el) {
-	exts[el] = true;
-});
+const exts = new Set(binaryExtensions);
 
-module.exports = function (filepath) {
-	return path.extname(filepath).slice(1).toLowerCase() in exts;
+module.exports = filepath => {
+	return exts.has(path.extname(filepath).slice(1).toLowerCase());
```
Comment (sindresorhus) [inline index.js]: Make it an inline arrow function. No need for `return`.

File: readme.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ $ npm install --save is-binary-path
 ## Usage
 
 ```js
-var isBinaryPath = require('is-binary-path');
+const isBinaryPath = require('is-binary-path');
```
Comment (sindresorhus) [inline readme.md:16]: Can you also update `http://sindresorhus.com` to https and remove the backticks from the Related links?

Comment (kevva) [reply readme.md:16]: Certainly. I intentionally kept unrelated changes like those outside of this PR. You want me to add `xo` too?

Comment (sindresorhus) [reply readme.md:16]: Sure, that would be lovely.

**Comment by sindresorhus** [comment]
Wonderful :)

**Pull request merged**